components/_patterns/00-base/global/base/_03-mixins.scss

```diff
@@ -0,0 +1,50 @@
+/////////////////////
+// General Mixins (put specific ones in component files where applicable)
+
+/// Mixin - Clearfix.
+/// Adds clearfix based on http://bourbon.io/docs/#clearfix
+/// use example =   @include cleafix
+
+@mixin clearfix {
+  &::after {
+    clear: both;
+    content: "";
+    display: table;
+  }
+}
+
+/// Mixin - Wrapper
+/// Outer container mixin for large screens
+@mixin wrapper($container-max-width: $max-width, $outer-container-break: $small, $v-margin: 0, $v-padding: 0, $h-padding: $space, $h-padding-large: $h-padding) {
+  max-width: #{$container-max-width};
+  width: 100%;
+  margin: #{$v-margin} auto;
+  padding: #{$v-padding} #{$h-padding};
+
+  @include breakpoint($outer-container-break) {
+    padding: #{$v-padding} #{$h-padding-large};
+  }
+
+  @include breakpoint($container-max-width) {
+    padding-left: calc(#{$h-padding-large} + calc(-50vw + calc(#{$container-max-width} / 2)));
+    padding-right: calc(#{$h-padding-large} + calc(-50vw + calc(#{$container-max-width} / 2)));
+  }
+}
+
+/// Use the breakout mixin for elements that should be edge-to-edge
```

Comment:
What do you think about separating all the full-width pieces out into a gist? I like the code, but this is really a project-specific need and makes the wrapper mixin a little less approachable.

Reply:
The breakout mixin doesn't affect the wrapper mixin in any way. The wrapper mixin will stay exactly the same, even if we remove the breakout mixin.

I'd like to keep the breakout in here. Maybe I just need to record a quick "Here's how to use this" video, or write a blogpost.

I don't think it's project-specific, it's just "any time you have a broad-context container (like `body` or `main`) and you need a way to have a full-width element, this is how you do it.

Reply:
No worries - it was a minor nitpick to begin with. More prescriptive than my taste for Emulsify "core," but you're right about it not simplifying the breakout mixin.  I was thinking we could get rid of some of the breakout mixin args, but you're using them all anyway.

I don't think a post/video is necessary for such a small piece of code (although showing off how it was used on a client project might be cool). It might not hurt to add some CSS comments or a link to someone else's post in the code (not necessary but maybe helpful for someone who hadn't come across that technique).